### PR TITLE
style: Format MDX files with Prettier.

### PR DESCRIPTION
**Pull request by avocadowastaken**
Replacement for the #1995.

### CODE REVIEW

**Comment by janpio** [comment]
Using Percy for this is _amazing_ - I left comments on the view pages where something weird happened. (Can I make these review visible here in GH somehow?)

**Comment by avocadowastaken** [comment]
> Using Percy for this is _amazing_ - I left comments on the view pages where something weird happened. (Can I make these review visible here in GH somehow?)

I know only this one, so far it was enough for me :)

image

**Comment by janpio** [comment]
Oh nice, I did not notice that. Sweet.

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/index.mdx [hunk 1]
```diff
@@ -22,8 +22,9 @@ In order to successfully complete this guide, you need:
 - Access to a MongoDB 4.2+ server with a replica set deployment. We recommend using [MongoDB Atlas](https://www.mongodb.com/cloud/atlas).
 
   <Admonition type="warning">
-  
- The MongoDB database connector uses transactions to support nested writes. Transactions **requires** a [replica set](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) deployment. The easiest way to deploy a replica set is with [Atlas](https://docs.atlas.mongodb.com/getting-started/). It's free to get started.
+
+  The MongoDB database connector uses transactions to support nested writes. Transactions **requires** a [replica set](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) deployment. The easiest way to deploy a replica set is with [Atlas](https://docs.atlas.mongodb.com/getting-started/). It's free to get started.
```
Comment (avocadowastaken) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/index.mdx:26]: Looks like it fixed markup in https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/mongodb-typescript-mongodb#prerequisites

It's looks like this now

image

File: content/200-concepts/200-database-connectors/03-postgresql.mdx [hunk 2]
```diff
@@ -180,31 +180,31 @@ When introspecting a PostgreSQL database, the database types are mapped to Prism
...
-| `jsonb`                                     | `Json`        |    ✔️     | `@db.JsonB`\*                            | \*Default mapping for `Json` - no type attribute added to schema.     |
-| `bytea`                                     | `Bytes`       |    ✔️     | `@db.ByteA` \*                           |                                                                       | \*Default mapping for `Bytes` - no type attribute added to schema. |
+| `jsonb`                                     | `Json`        |    ✔️     | `@db.JsonB`\*                            | \*Default mapping for `Json` - no type attribute added to schema.                                                                                                               |
+| `bytea`                                     | `Bytes`       |    ✔️     | `@db.ByteA` \*                           |                                                                                                                                                                                 | \*Default mapping for `Bytes` - no type attribute added to schema. |
```
Comment (avocadowastaken) [inline content/200-concepts/200-database-connectors/03-postgresql.mdx]: Looks like this description is on 6th column, but should be on 5th, and therefore is not visible now now http://localhost:8000/concepts/database-connectors/postgresql/#native-type-mappings

Looks like that: 

image

Prettier normalized table and add 1 extra column for each of row, and now looks like that:

image

**Comment by avocadowastaken** [comment]
@janpio I couldn't reply in the percy.io, so I add comments here.

Also, not sure why some text got screwed, or some images not loading properly. Maybe it's worth to ask them by Intercom?

File: content/200-concepts/200-database-connectors/03-postgresql.mdx [hunk 3]
```diff
@@ -180,31 +180,31 @@ When introspecting a PostgreSQL database, the database types are mapped to Prism
...
 | `json`                                      | `Json`        |    ✔️     | `@db.Json`                               |
-| `jsonb`                                     | `Json`        |    ✔️     | `@db.JsonB`\*                            | \*Default mapping for `Json` - no type attribute added to schema.     |
-| `bytea`                                     | `Bytes`       |    ✔️     | `@db.ByteA` \*                           |                                                                       | \*Default mapping for `Bytes` - no type attribute added to schema. |
+| `jsonb`                                     | `Json`        |    ✔️     | `@db.JsonB`\*                            | \*Default mapping for `Json` - no type attribute added to schema.                                                                                                               |
+| `bytea`                                     | `Bytes`       |    ✔️     | `@db.ByteA`\*                            | \*Default mapping for `Bytes` - no type attribute added to schema.                                                                                                              |
```
Comment (janpio) [inline content/200-concepts/200-database-connectors/03-postgresql.mdx:184]: Something changed here I think.

**Comment by janpio** [comment]
Added an issue for the image problem: https://github.com/prisma/docs/issues/2041

**Comment by janpio** [comment]
Wheeee @umidbekk!

**Pull request merged**